test/verify/check-system-info

```diff
@@ -281,12 +284,18 @@ class TestSystemInfo(MachineCase):
         self.assertIn("Mon Jun  4 06:34:", m.execute("date"))
         self.assertIn("EEST 2018\n", m.execute("date"))
 
-    def testTimeServers(self):
+    @skipImage("timesyncd not available", "rhel-9-2", "rhel-8-8", "centos-8-stream")
+    @skipImage("Need a mutable OS", "fedora-coreos")
```

Comment:
and "rhel4edge" since yesterday. But configuring NTP isn't related to mutable /usr, this shouldn't touch anything in /usr? Fedora CoreOS already uses timesyncd by default, so no package needs to be removed. rhel4edge is presumably using chrony, skipping the test there is correct.

Reply:
The test itself wants to mutate the OS image.  I try to make it work differently.

Reply:
Done.